shared-bindings/alarm/__init__.c

```diff
@@ -199,7 +199,18 @@ STATIC mp_map_elem_t alarm_module_globals_table[] = {
     { MP_ROM_QSTR(MP_QSTR_SleepMemory),   MP_OBJ_FROM_PTR(&alarm_sleep_memory_type) },
     { MP_ROM_QSTR(MP_QSTR_sleep_memory),  MP_OBJ_FROM_PTR(&alarm_sleep_memory_obj) },
 };
-STATIC MP_DEFINE_MUTABLE_DICT(alarm_module_globals, alarm_module_globals_table);
+MP_DEFINE_MUTABLE_DICT(alarm_module_globals, alarm_module_globals_table);
```

Comment:
I deleted the `STATIC` here by mistake, during some code evolution on the code fix.